File: Programming-languages-and-laboratory/Java/Lessons/34_20221221/29-soluzione-a/Pyhton/servizio.py
from dipendente import Dipendente

class Servizio:
    dipendente: Dipendente
    numero_ore: float

    def __init__(self, dipendente: Dipendente, numero_ore: float):
        self.dipendente = dipendente
        self.numero_ore = numero_ore

    def get_costo(self) -> float:
        return self.numero_ore * self.dipendente.costo_orario

File: II - Linguaggi di programmazione e laboratorio/Java/Exercises/29_compito_java_2019_04_11a/Servizio.java
public class Servizio {
	protected Dipendente dip;
	protected double oreServizio;

	public Servizio(Dipendente dip, double oreServizio) {
		this.dip = dip;
		this.oreServizio = oreServizio;
	}

	public double getValore() {
		return oreServizio * dip.getCosto();

	}
}